e2e-tests/playwright/docs/accessibility/automated_scan_testing.md

```diff
@@ -0,0 +1,145 @@
+# Automated Accessibility Scan Testing Guidelines
+
+## Overview
+
+This guide focuses on **automated accessibility testing** using a **simplified, comprehensive approach**. Use axe-core's default rule set and scope testing to specific elements, disabling rules only when necessary. This maximizes coverage while minimizing maintenance overhead.
+
+## Testing Philosophy
+
+### **Core Principle: Scope and Analyze**
+
+Use axe-core's comprehensive default rule set and scope testing to specific elements.
+
+**Benefits:**
+
+- **Maximum Coverage** - All WCAG 2.1 AA rules by default
+- **Low Maintenance** - No rule lists to maintain
+- **Future-Proof** - New axe-core rules automatically included
+- **Clear Intent** - Disabled rules are explicit and documented
+
+### **Basic Pattern**
+
+```typescript
+test('page accessibility', async ({axe, page}) => {
+    const results = await axe.builder(page).analyze(); // All applicable rules
+
+    expect(results.violations).toHaveLength(0);
+});
+```
+
+```typescript
+test('component accessibility', async ({axe, page}) => {
+    const results = await axe
+        .builder(page)
+        .include('#element') // Scope to element
+        .analyze(); // All applicable rules
+
+    expect(results.violations).toHaveLength(0);
+});
+```
+
+## What Can Be Automated
+
+### **Fully Automatable (90-95% Coverage)**
+
+- **HTML Structure & Semantics** - Markup validity, semantic elements, heading hierarchy
+- **ARIA Implementation** - Attributes, states, relationships, roles
+- **Form Accessibility** - Labels, validation, error messages
+- **Interactive Elements** - Focus capability, accessible names
+- **Color and Contrast** - WCAG AA/AAA contrast ratios
+- **Alternative Text** - Image alt text presence
+- **Table Structure** - Headers, captions, relationships
+- **Navigation** - Skip links, landmarks, focus order
+
+### **Cannot Be Fully Automated**
+
+- **Cognitive Accessibility** - Content readability, comprehension
+- **Contextual Appropriateness** - Alt text quality, meaningful link text
+- **Real User Experience** - Actual keyboard workflows, screen reader UX
+- **Content Quality** - Clear instructions, effective error messaging
+
+## Testing Patterns
+
+### **1. Component Testing**
+
+```typescript
+test('modal accessibility', async ({axe, page}) => {
+    await page.click('[data-testid="open-modal"]');
```

Comment:
Could you change these to address elements using accessible selectors if possible? I think it would be good to discourage `data-testid` in favour of getting elements by their text or label